modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetCategoryLocalServiceTest.java

```diff
@@ -629,6 +633,32 @@ public void testGetCategoriesWithoutPermissions() throws Exception {
 		}
 	}
 
+	@Test(expected = NoSuchCategoryException.class)
```

Comment:
We did like this in the first instance (so multiple tests) but Brian changed to a single one with comments inside. Could you please have a look to the RoleLocalServiceTest about these?

Reply:
I saw it and I did prefer this way... 

but it Brian asked for it I will change it 😞